src/Riok.Mapperly/Emit/Syntax/SyntaxFactoryHelper.Invocation.cs

```diff
@@ -123,6 +123,12 @@ public static string StaticMethodString(IMethodSymbol method)
         return $"{qualifiedReceiverName}.{method.Name}";
     }
 
+    private ArgumentListSyntax LineSeparatedArgumentList(IEnumerable<ExpressionSyntax> argSyntaxes) =>
+        SyntaxFactory.ArgumentList(ConditionalCommaLineFeedSeparatedList(argSyntaxes.Select(Argument)));
+
+    private ArgumentListSyntax LineSeparatedArgumentList(IEnumerable<ArgumentSyntax> argSyntaxes) =>
+        SyntaxFactory.ArgumentList(ConditionalCommaLineFeedSeparatedList(argSyntaxes));
+
     private static ArgumentListSyntax ArgumentList(params ExpressionSyntax[] argSyntaxes) =>
         SyntaxFactory.ArgumentList(CommaSeparatedList(argSyntaxes.Select(Argument)));
 
```

Comment:
The method `ArgumentList(IEnumerable<ArgumentSyntax> args)` is now unused, isn't it?